ruby_on_rails/apis/apis_and_building_your_own.md

```diff
@@ -140,25 +140,25 @@ Here's a simple example (again we are just rendering the error in all cases):
 ~~~
 
 
-#### Creating Dynamic Error Pages
+#### Creating dynamic error pages
 
 You can create your own error pages. See [this post](https://web-crunch.com/posts/custom-error-page-ruby-on-rails).
 
 Sometimes Heroku can require additional steps to properly display your error pages.  See [their error page docs here](https://devcenter.heroku.com/articles/error-pages).  You might need to delete the static pages in the `app/public` directory first.
 
-#### External Facing Security
+#### External facing security
 
 Let's say you want to only allow an API call if your user is logged in.  Your existing controller authentication will work to cover this as well -- just make sure you've got the right `#before_action` set up (e.g. `before_action :require_login`).  This might be the case if both logged in and non-logged-in users will be viewing the page but each should see different data.  You don't want your not-logged-in-users to be able to make API requests for sensitive data just like you wouldn't want them to be able to visit an unauthorized HTML page.
 
 <span id="api-tokens">If you want to handle requests from an application that isn't a web browser (e.g. the command line), you can't rely on browser cookies to authenticate you.  That's why most APIs issue custom tokens to each authorized user which must be sent along with the request as part of the authentication process.</span>  We'll talk a bit more about tokens in the next lesson.
 
-#### Next Steps
+#### Next steps
 
 Right now you've got the ability to use your Rails app to serve up not just HTML but also whatever formats you want.  If you'd like to take this further and allow other developers to build on your platform (so they'll be making programmatic requests instead of logging in as users), you'll need to make your API system far more robust.  We won't cover it all here, but check out the following:
 
 * [Building Awesome Rails APIs](http://collectiveidea.com/blog/archives/2013/06/13/building-awesome-rails-apis-part-1/) has a lot of best practices for moving beyond a toy API and into a production-ready app.
 
-### Service Oriented Architecture (SOA)
+### Service oriented architecture (SOA)
```

Comment:
```suggestion
### Service Oriented Architecture (SOA)
```

This is a tricky one since it's used in different ways online. I think we can leave it capitalised.

Reply:
Thanks! I remember I couldn't decide here because I saw it's not capitalised online